app/interfaces/api/v1/advocates/fee.rb

```diff
@@ -21,6 +21,7 @@ class Fee < Grape::API
               requires :claim_id, type: Integer
               requires :fee_type_id, type: Integer
               requires :quantity, type: Integer
+              requires :amount, type: Float
```

Comment:
:amount needs to be added into #args (should it's presence also be validated on the model?)